tests/unit_tests/sym_functions.cpp

```diff
@@ -215,15 +216,16 @@ TEST_CASE("Log") {
     sym logC = sym(sympp::log(C, C));
     sym logD = sym(sympp::log(D, D));
 
-    // REQUIRE(logA == logB);
-    // REQUIRE(logA != logC);
+    REQUIRE(logA == logB);
+    REQUIRE(logA != logC);
 
-    // REQUIRE(logA.evaluate(bool_val, int_val, doub_val) ==
-    // logB.evaluate(bool_val, int_val, doub_val));
+    // REQUIRE(logA.evaluate(bool_val, int_val, doub_val) ==   logB.evaluate(bool_val, int_val, doub_val));
 
+    // Com erro
     REQUIRE(logA.evaluate_sym(bool_val, int_val, doub_val) == logA);
```

Comment:
Tente enviar só o código que passar nos unit tests e vá corrigindo na medida que cria novos testes. O que for comentário pessoal, favor remover antes de enviar ao repositório. O que for comentário relevante, como um bug que descobriu mas não vai corrigir agora, aí sim pode escrever que é um erro e deixar a linha que causaria o erro comentada.